nixos/tests/podman/base.nix

```
@@ -0,0 +1,7 @@
+{ system ? builtins.currentSystem, pkgs ? import ../../.. { inherit system; } }:
+
+{
```

A `recurseIntoAttrs` here has a good chance of fixing the behavior in ofborg.

```suggestion
pkgs.lib.recurseIntoAttrs {
```

---

Seems it half works, `ofborg` must handle build and test differently.

I'll dig around in the code tomorrow.

---

Forgot to post a follow up comment:

Build and test does seem to be handled differently.

While it would be nice to have, with the rather glacial pace `ofborg` development, the possibility "fixing" it breaks other tests and the testing RFC that may necessitate changes anyway, I don't really see much point in continuing with this for now.